tests/TestGroup.cpp

```diff
@@ -1052,12 +1053,12 @@ void TestGroup::testChildrenSort()
 
 void TestGroup::testHierarchy()
 {
-    Group* group1 = new Group();
-    group1->setName("group1");
+    Group group1;
+    group1.setName("group1");
 
     Group* group2 = new Group();
     group2->setName("group2");
-    group2->setParent(group1);
+    group2->setParent(&group1);
```

Comment:
This will lead to double free. Do not add groups to the DB that are allocated on the stack.

Reply:
No, because the group isn't added to a database in this test case.

Reply:
Ok